pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -103,6 +103,17 @@ def restore(self, checkpoint_path: str, on_gpu: bool):
         # load model state
         model = self.trainer.get_model()
 
+        # restore states
+        self.restore_model_states(model, checkpoint)
+        if on_gpu:
+            model.cuda(self.trainer.root_gpu)
+        self.restore_training_state(checkpoint)
+
+    def restore_model_states(self, model: LightningModule, checkpoint) -> None:
```

Comment:
```suggestion
    def restore_model_state(self, model: LightningModule, checkpoint) -> None:
```

Reply:
Acutually, `restore_model_states` restore **multiple** state**s** (nn.Module, dataloader, AMP).  
This is the reason why I name it with plural form.  

When I first dived into save/load codes, I miss-interpreted that "hmm, `restore_training_state` (※training) may restore single training state, like PyTorch's `load_state_dict`".  
This is the background I use above naming convention.  

Which do you think better?

Reply:
I think `restore_training_state` and `restore_model_state` is good enough. But no strong opinion though :)